docs/source/pillars/information_governance.md

```diff
@@ -86,6 +86,12 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You report on and share outcomes of each audit of your TRE organisation with the required bodies
   - This may included regulatory bodies or the organisations that manage accreditations you have
   - Mandatory
+* - You log user actions, network connection attempts, infrastructure actions and data access within a TRE
+  - The greater the number of (and diversity of) actions logged, the higher the chance that any bad behaviours, intentional or unintentional, are spotted.
+    Bad behaviours in this context are anything that could compromise the security of the SRE, or result in the accidental discolure of data outside the TRE.
```

Comment:
Is this an exhaustive list of bad behaviours?

Reply:
My hope here is that a TRE developer adopting the standard should determine for themselves what exactly the behaviours themselves are, but the principle is to log anything they feel could be useful - maybe this is too wishy washy? But I'm not sure whether it should be too prescriptive either